css3-selectors/README.md

```diff
@@ -1,17 +1,18 @@
-# Advanced CSS: CSS3 Selectors
+# CSS3 Selectors
 
-In this workshop, you will learn modern CSS3 selectors that will let you write more powerful, specific CSS rules. You'll strengthen your CSS rule writing muscles, helping you be more comfortable in the <style> tag.
+Want to style the right element(s) – and only the right element(s)? CSS3 selectors are here for you.
 
-The workshop will be a mix of lectures and hands-on exercises in pairs. It will be taught by {{ teacher_name }}, {{ teacher_description }}.
+In this workshop, you will gain a solid understanding of modern CSS3 selectors. These will let you write more powerful, specific CSS rules. You'll strengthen your CSS rule writing muscles, helping you be more comfortable in the `<style>` tag.
+
+The workshop will be a mix of lectures and hands-on exercises. It will be taught by `{{ teacher_name }}`, `{{ teacher_description }}`.
 
 This is a great follow-up workshop for those who attended our Intro to HTML & CSS workshop.
 
 **Topics covered**
 
-- CSS attribute selectors
-- CSS3 form pseudo-classes
-- CSS3 structural selectors
-- CSS3 pseudo-elements
+- **Review of CSS relational selectors:** How to select and style elements of the page based on where they are in relation to other elements. You'll learn how to do this as specifically or generally as you want (all images, all images in lists, all images in a specific list).
```

Comment:
These descriptions are great. Super detailed and really helpful. Nice work on them!